pkg/controller/sync/sync_controller.go

```diff
@@ -216,6 +230,19 @@ func (r *ReconcileSync) Reconcile(request reconcile.Request) (reconcile.Result,
 	return reconcile.Result{}, nil
 }
 
+func (r *ReconcileSync) skipExcludedNamespace(namespace string) bool {
+	r.operationExcluder.Mux.Lock()
+	defer r.operationExcluder.Mux.Unlock()
+
+	if r.operationExcluder != nil {
+		excludedNS := r.operationExcluder.GetExcludedNamespaces(match.Sync)
+		if len(excludedNS) > 0 {
```

Comment:
Is this guarding against nil? Otherwise checking the length should be unnecessary, as the default of `false` will be returned.

Reply:
i think this was a leftover from previous map, removed now